### DISCUSSION TITLE
Updating SpotX

### DISCUSSION THREAD

**Discussion by ghost** [SpotX-Official/SpotX#272] [Question]
Greetings!

This may be a foolish or novice question for more experienced coders and developers like yourself, but I was curious as to how I should go about updating SpotX? I know that automatic Spotify updates are disabled lest SpotX's programming is overwritten, but I am unclear as to how to update SpotX's own updates and patches published and dispersed by the developers on here.

Your consideration of this inquiry is greatly appreciated. Apologies if this question was aforementioned by others!

**Comment by amd64fox** [comment]
to update SpotX you can simply run the same installation you ran before.

**Reply by ghost** [reply]
As in reinstalling it?

**Reply by amd64fox** [reply]
yeah

**Reply by ghost** [reply]
Ah, alright. Apologies for my tech-"unsavviness." Thanks for your assistance!
